datafusion/core/src/physical_plan/common.rs

```diff
@@ -109,9 +109,13 @@ pub(crate) fn spawn_buffered(
             builder.spawn(async move {
                 while let Some(item) = input.next().await {
                     if sender.send(item).await.is_err() {
```

Comment:
The only reason an item can't be sent via a channel I think is if the other end (the `Receiver`) was dropped. This can certainly happen if some other part of the query errors, but I also think it can happen with the plan shuts down early due to a `LIMIT` or something similar. 

Thus I am not sure we should propagate an error in this case

Reply:
Makes sense. Removed the error.